pkg/tf2pulumi/convert/tf.go

```diff
@@ -935,6 +935,16 @@ func camelCaseName(name string) string {
 	return name
 }
 
+// Returns whether the fully qualified path is being applied for a property
+// for which we can get the `maxItemsOne` field.
```

Comment:
I'd remove this second line of the comment. Presumably this isn't unique to "maxItemsOne" and we may look up other attributes of properties in the future.

Reply:
Done ✅